includes/class-edd-html-elements.php

```diff
@@ -524,28 +527,33 @@ public function year_dropdown( $name = 'year', $selected = 0, $years_before = 5,
 	 *
 	 * @since 1.5.2
 	 *
-	 * @param string $name     Name attribute of the dropdown
-	 * @param int    $selected Month to select automatically
+	 * @param string  $name     Name attribute of the dropdown.
+	 * @param int     $selected Month to select automatically.
+	 * @param string  $id A unique identifier for the field.
+	 * @param boolean $return_long_name Whether to use the long name for the month.
```

Comment:
Same as above -- let's line up the descriptions with each other. It makes it a little easier to read.
(For both `$id` and `$return_long_name`)